README.md

```diff
@@ -22,15 +22,15 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 <br/>
 
-# Welcome! 3 Things You Ought To Know First:
+# Welcome! First, three things you should know:
 
-**1. When you read here, you in fact read dozens of the best Node.js articles -** this is a summary and curation of the top-ranked content on Node.js best practices
+**1. The content presented here is a curation of the best Node.js articles -** this repository is a summary and curation of the top-ranked content on Node.js best practices, as well as content written here by collaborators
```

Comment:
nit: The idea of 'curation' is repeated in the bullet and its description. The original text, in spirit, seemed more 'DRY'.

```suggestion
**1. You are, in fact, reading dozens of the best Node.js articles -** this repository is a summary and curation of the top-ranked content on Node.js best practices, as well as content written by collaborators
```